server/src/test/java/com/broadcom/lsp/cobol/domain/databus/impl/DataBusUnSubscribeTest.java

```diff
@@ -52,14 +50,12 @@ void subscribe() {
         TimeoutException.class,
         () -> {
           // Subscribe
-          Object subscriber = databus.subscribe(DataEventType.REQUIRED_COPYBOOK_EVENT, this);
-          databus.postData(
-              RequiredCopybookEvent.builder().name("CPYBUILD_SUBSCRIPTION TEST").build());
+          Object subscriber = databus.subscribe(DataEventType.RUN_ANALYSIS_EVENT, this);
+          databus.postData(new RunAnalysisEvent());
           waiter.await(5000);
```

Comment:
Can we define a meaningful const, why we wait, it is hardcoded multiple times as well. I know old code ;)

Reply:
Why not? It is refactoring!